### PR TITLE
Make code samples copy+paste friendly

### LINKED ISSUE #723: Remove $ from bash code samples
The starting $ in shell code samples doesn't play well with the ability to copy + paste code samples.

These should be removed, and while we're at it we should also remove server responses, and anything else that stops users from simply copy + pasting code into a terminal window.

### LINKED ISSUE #731: Fix the copy button
or the code examples themselves because now when you clip the copy button, it copies also the prompt sign and the result from the command. As an example, if I copy the content of this image

image

I get

```
$ curl -L https://install.meilisearch.com | sh
$ ./meilisearch
Server is listening on: http://127.0.0.1:7700
```

instead of the desired outcome

```
curl -L https://install.meilisearch.com | sh
./meilisearch
```

### PR DESCRIPTION
**Pull request by react-learner**
Remove command prompt from bash code samples, separate input from server response.
Fixes #731 , fixes #723

### CODE REVIEW

**Review by fharper — APPROVED** [review]
LGTM

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
Small details! You can ignore them if they are not useful for you :)

File: create/how_to/running_production.md [hunk 1]
```diff
@@ -101,7 +100,7 @@ Service files are text files that tell your operating system how to run your pro
 To run MeiliSearch in a production environment, use the `--env` flag. To generate a master key that will let MeiliSearch create reading and writing keys, use the `--master-key` flag. With those keys, you can easily control who can access or create new documents, indexes, or change the configuration. You can change the `Master Key` to any value in the following command. However, for security concerns, it's better to choose a safe and random key, never share it and, just, **keep it safe**.
 
 ```bash
-$ cat << EOF >/etc/systemd/system/meilisearch.service
+cat << EOF >/etc/systemd/system/meilisearch.service
 [Unit]
 Description=MeiliSearch
 After=systemd-user-sessions.service
```
[the remark this answers is not in the document]
Comment (react-learner) [reply create/how_to/running_production.md:106]: Not exactly, I think cat is being used to write standard input to a file (called a [here document](https://en.wikipedia.org/wiki/Here_document)).
But anyway it doesn't work great with copy+paste, so it should be changed. I'm going to ask @eskombro about it.

[the remark this answers is not in the document]
Comment (react-learner) [reply create/how_to/running_production.md:106]: Nevermind, it works with copy and paste and doesn't need to be changed.

File: reference/features/installation.md [hunk 2]
```diff
@@ -10,8 +10,10 @@ Download the **latest stable release** of MeiliSearch with **curl**.
 Launch MeiliSearch to start the server.
 
 ```bash
-$ curl -L https://install.meilisearch.com | sh
-$ ./meilisearch
+curl -L https://install.meilisearch.com | sh
+./meilisearch
+```
+```bash
 Server is listening on: http://127.0.0.1:7700
```
[the remark this answers is not in the document]
Comment (react-learner) [reply reference/features/installation.md:18]: No that's a good idea, I'll fix it.

File: reference/features/installation.md [hunk 3]
```diff
@@ -23,8 +25,10 @@ Download the **latest stable release** of MeiliSearch with **Homebrew**.
 Launch MeiliSearch to start the server.
 
 ```bash
-$ brew update && brew install meilisearch
-$ meilisearch
+brew update && brew install meilisearch
+meilisearch
```
[the remark this answers is not in the document]
Comment (react-learner) [reply reference/features/installation.md:30]: Let's do it in another PR.

File: reference/features/installation.md [hunk 4]
```diff
@@ -53,9 +59,11 @@ Download the **latest stable release** of MeiliSearch with **APT**.
 Launch MeiliSearch to start the server.
 
 ```bash
-$ echo "deb [trusted=yes] https://apt.fury.io/meilisearch/ /" > /etc/apt/sources.list.d/fury.list
-$ apt update && apt install meilisearch-http
-$ meilisearch
+echo "deb [trusted=yes] https://apt.fury.io/meilisearch/ /" > /etc/apt/sources.list.d/fury.list
+apt update && apt install meilisearch-http
+meilisearch
```
[the remark this answers is not in the document]
Comment (react-learner) [reply reference/features/installation.md:67]: Oh, I didn't see that you wrote this as well. I'll add the two that you wrote.

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**